components/eventing-controller/api/v1alpha1/subscription_types_unit_test.go

```diff
@@ -1,4 +1,5 @@
-package v1alpha1
+// todo unit
```

Comment:
what about this todo ?

Reply:
In the next step, after all the test files are restructured, we will a build tags like:
```
// +build integration
```
so we can run unit, component and integration separately.
The todo helps to define what tag there should be.

The original plan was to do it all in one big pr at the same time but that makes debugging too hard and messy.